src/trans/gpu/internal/fsc_mod.F90

```diff
@@ -182,6 +183,7 @@ SUBROUTINE FSC(ALLOCATOR,HFSC,PREEL_COMPLEX, KF_FS, KF_UV, KF_SCALARS, KUV_OFFSE
 
 !*       2.1      U AND V.
 
+ILOEN_MAX = MAXVAL(G_NLOEN)
```

Comment:
Do we need to `COPY` this in for the parallel region below?

Reply:
yes, I think so, either we have to drop DEFAULT(NONE), then it is firstprivate, or we have to add it explicitly, otherwise it won't compile on Cray (which I think is the standard compliant way)

Reply:
Okay, let me suggest the change so you can consider it for merging.

Reply:
fixed, will commit it in a moment.